### ISSUE TITLE
Error running command: dotnet lambda list-layers

### ISSUE THREAD

**Issue by divyarajd7** [aws/aws-extensions-for-dotnet-cli#90]
I am able to successfully run this command 
```sh
dotnet lambda publish-layer --package-manifest LambdaLayer/packages.csproj rmoFMSLayer --layer-type runtime-package-store --s3-bucket bucketname --framework netcoreapp2.1
````
The layer does get uploaded correctly

However when I try to list the lambda layers, I get this error.
I did not have this issue 12 days ago, pushing the last successfully deployed build also now fails with this error.
![layerError](https://user-images.githubusercontent.com/35204699/64630202-95b08300-d3c2-11e9-8fd1-89d9c0adbb48.JPG)

Can you please provide guidance if I am doing anything wrong?
Any help appreciated.

**Comment by klaytaybai** [comment]
Thanks for reporting this.

I'm curious though why you are only seeing this issue now and not 12 days ago.  Did you change versions of Lambda Tools or make changes to the manifest or description?

**Comment by normj** [comment]
I found the problem and checked in a fix. https://github.com/aws/aws-extensions-for-dotnet-cli/commit/c2c44411cf888b4b5bbec175009f05898a0b0572

I'll get this fix shipped out as soon as I can.

**Comment by normj** [comment]
Sorry didn't mean to close the issue till fix is released.

**Comment by normj** [comment]
The fix for this has been shipped as part of version 3.3.1 of Amazon.Lambda.Tools 3.3.1
